.github/workflows/test-ci.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         pip3 install torch torchvision
         pip install -e .
         pip install "gym[atari,accept-rom-license]"
+        pip install -e ".[mujoco]"
```

Comment:
Don't do this on a separate line. It is enough to have one line `pip install -e .[mujoco]`

Also you can put your atari dependencies to setup.py to [atari] section and then you can just have one line instead of 3:
`pip install -e .[atari,mujoco]`

Reply:
I'll do it.